app/controllers/api_controller/vms.rb

```diff
@@ -150,6 +150,19 @@ def retire_resource_vms(type, id = nil, data = nil)
       end
     end
 
+    def restart_resource_vms(type, id = nil, _data = nil)
```

Comment:
:+1:, don't know why but I thought the API actions were out of bounds. Will change (and guest shutdown) when I rebase each in turn. Thanks!